saml/models/core/response_test.go

```diff
@@ -81,12 +81,12 @@ func Test_ParseResponse_Assertion(t *testing.T) {
 	t.Parallel()
```

Comment:
Going to restructure these tests and add more coverage if this PR looks reasonable.

Reply:
Done in https://github.com/hashicorp/cap/pull/104/commits/4655a4b4b988384b63f33b722ba7f3da55f9fbdd